.github/workflows/dockerimage.yml

```diff
@@ -17,20 +17,24 @@ jobs:
       env:
           DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
           DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
-      run: docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD 
+      run: docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+
+    - name: Tags
+      id: tags
+      uses: livepeer/action-gh-release-tags@v0
```

Comment:
`v0` here is a tag we can just keep updating to move these forward, yeah? It's not pinned or something?

Reply:
Yeah! So we can just move the tag on the action repository and it will start using the latest version it's pointing to ✨